libshpool/src/daemon/server.rs

```diff
@@ -197,6 +197,7 @@ impl Server {
 
         let (child_exit_notifier, inner_to_stream, pager_ctl_slot, status) = {
             // we unwrap to propagate the poison as an unwind
+            let _s = span!(Level::INFO, "1_lock(shells)").entered();
```

Comment:
why is this different from the other `lock(shells)`? what is the significance of the preceding `1` in this case?  Should use a word to describe it? (maybe even `first_lock(shells)`)

Reply:
There are two points were we lock shells in this routine, so to differentiate the scopes I wanted to prefix them with something. I couldn't think of a great name here so I just used numbers.

Reply:
sg! :)